config/routes.rb

```diff
@@ -5,3 +5,7 @@
 put 'projects/:id/settings/gtt',
   to: 'projects#update_gtt_configuration',
   as: :update_gtt_configuration
+
+scope 'gtt' do
+  get 'settings', to: 'gtt_configuration#default_setting_configuration', as: :default_setting_configuration
+end
```

Comment:
@nobnisai 
This is a slight thing, but ending with a new line may be better for compare diff.